spring-cloud-netflix-eureka-server/src/main/java/org/springframework/cloud/netflix/eureka/server/EurekaController.java

```diff
@@ -288,13 +288,14 @@ protected void filterReplicas(Map<String, Object> model, StatusInfo statusInfo)
 	private String scrubBasicAuth(String urlList){
 		String[] urls=urlList.split(",");
 		String filteredUrls="";
+		StringBuilder sb = new StringBuilder();
```

Comment:
Remove the `String` variable and change the `StringBuilder` to `filteredUrls`

Reply:
Thank you for your reply, I will modify it according to your suggestion : )